wisdem/inputs/modeling_schema.yaml

```diff
@@ -215,6 +215,17 @@ properties:
                         maximum: 5.0
                         default: 1.0
                         unit: none
+                    use_gb_torque_density:
+                        type: boolean
+                        default: True
+                        description: Flag switching between running to gearbox sizing of DrivetrainSE (False) or the simple sizing given a value of torque density expressed in Nm/kg
+                    gearbox_torque_density:
```

Comment:
Curious why this second one is in modeling_schema and not the geometry_schema?  I guess you could make an argument either way.  It gets loaded into a regular OpenMDAO input though and doesn't really get used like a user-option.

Reply:
I had it in modeling because it was next to the flag, but in geometry makes more sense, thank you for the suggestion